docs/en_US/NAS/ClassicNas.md

```diff
@@ -0,0 +1,33 @@
+# Classic NAS Algorithms
+
+In classic NAS algorithms, each architecture is trained as a trial and the NAS algorithm acts as a tuner. Thus, this training mode naturally fits within the NNI hyper-parameter tuning framework, where Tuner generates new architecture for the next trial and trials run in the training service.
```

Comment:
, where tuners generate ...

Reply:
I prefer to keep it. or change it to "where a tuner generates"